kong/cmd/utils/prefix_handler.lua

```diff
@@ -378,6 +378,54 @@ local function write_process_secrets_file(path, data)
   return true
 end
 
+local function pre_create_lmdb(conf)
+  local prefix = conf.prefix
+  local lmdb_path = conf.lmdb_environment_path or "dbless.lmdb"
+  local user = string.match(conf.nginx_user or "", "%w+") or "$USER"
+
+  local dir_name
+  if lmdb_path:sub(1, 1) == "/" then
+    dir_name = lmdb_path
+
+  else
+    dir_name = prefix .. "/" .. lmdb_path
+  end
+
+  if pl_path.isdir(dir_name) then
+    return true
+  end
+
+  log.info("LMDB directory '%s' does not exist, " ..
```

Comment:
```suggestion
  log.debug("LMDB directory '%s' does not exist, " ..
```

Log less by default, only log as `info` if it is something people should really care.